src/platform/datapath_raw_xdp_win.c

```diff
@@ -91,9 +91,9 @@ typedef struct XDP_QUEUE {
 } XDP_QUEUE;
 
 typedef struct DECLSPEC_ALIGN(MEMORY_ALLOCATION_ALIGNMENT) XDP_RX_PACKET {
-    CXPLAT_RECV_DATA;
-    CXPLAT_ROUTE RouteStorage;
     XDP_QUEUE* Queue;
+    CXPLAT_ROUTE RouteStorage;
+    CXPLAT_RECV_DATA RecvData;
```

Comment:
>CXPLAT_RECV_DATA RecvData; [](http://example.com/codeflow?start=4&length=26)

Have you made sure RecvData is still on a 16-byte alignment boundary?

Reply:
The first pointer sized member of `RecvData` is used as SLIST_ENTRY in `CxPlatDpRawRxFree`, which requires 16B alignment. Otherwise, the last 4 bits of that field isn't guaranteed to be zero.

Reply:
That's probably `RecvData` was the first member in the parent struct and the parent struct has an alignment requirement.